.github/workflows/publish.yml

```diff
@@ -117,10 +117,10 @@ jobs:
         run: |
           mkdir -p oso-js-node-${{ github.event.inputs.version }}
           tar -xzf oso-js-node-${{ github.event.inputs.version }}.tgz -C oso-js-node-${{ github.event.inputs.version }} --strip-components=1
-      - name: Use Node.js 12.20.0
+      - name: Use Node.js 10.24.0
         uses: actions/setup-node@v1
         with:
-          node-version: 12.20.0
+          node-version: 10.24.0
```

Comment:
Not consequential here, but a general note: I think YAML removes trailing `.0`s from numbers, so this will actually get parsed as `10.24` and the latest version that matches `10.24.x` will be installed instead of `10.24.0`.